app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments
+  has_many :likes
```

Comment:
Hello @PabloBona,

Great job on setting up the associations in your User model! However, I noticed a slight discrepancy between the project requirements and your implementation.

As per the ERD Diagram, the User has many posts, comments, and likes. it is specified that the foreign_key for all the has_many associations should be 'author_id'. In your code, you've correctly specified the foreign_key for has_many :posts, but missed doing so for has_many :comments and has_many :likes.


Remember, you're making good progress!  Keep up the good work!

Reply:
Thanks for your feedback.
Changes have already been implemented!

Reply:
@topeogunleye It wasn't good advice.  Is not author_id is user_id... I have got a lot of issues in spec...